nidirect_gp/src/Controller/GpController.php

```diff
@@ -25,8 +26,8 @@ class GpController extends ControllerBase implements ContainerInjectionInterface
    *   An array suitable for drupal_render().
    */
   public function revisionShow($gp_revision) {
-    $gp = $this->entityManager()->getStorage('gp')->loadRevision($gp_revision);
-    $view_builder = $this->entityManager()->getViewBuilder('gp');
+    $gp = \Drupal::entityTypeManager()->getStorage('gp')->loadRevision($gp_revision);
```

Comment:
Shouldn't call these statically, they should be injected as pre the previous code.

Reply:
Good spot. I suppose drupal-check hasn't got the full context of usage. Need to play with that a bit more to see if it's happy with that use case or if the syntax is triggering a false positive

Reply:
Fixed now: `s/$this->entityManager()/$this->entityTypeManager()/g`

:)

Reply:
Swapped out some extra static service calls for date formatting too. There are a handful more in there which were generated; there's a JIRA ticket to track that as it's taking me down a rabbithole now. The key issue on this PR of removing deprecated code looks resolved in any case